### ISSUE TITLE
Hotspot has old name, configure function returns error

### ISSUE THREAD

**Issue by sankar4n** [hypery2k/cordova-hotspot-plugin#21]
Hi Martin,

When I create a hotspot, the old name is being used for ssid, and the security is open even though I am setting it to `WPA_PSK` or `WPA2_PSK`. To work around this I am using the following code to start the default hotspot whatever the name is, then configuring it with my ssid and password. 

```
var q = $q.defer();
cordova.plugins.hotspot.startHotspot(function() {
  cordova.plugins.hotspot.configureHotspot(ssid, 'WPA_PSK', password, q.resolve, q.reject);
}, q.reject);
return q.promise;
```

when I run this, I am seeing following error

```
Attempt to invoke virtual method 'java.lang.String org.json.JSONArray.getString(int)' on a null object reference
```

I thought this might be related to #15 and updated the plugin to 1.0.0. I am still seeing the issue. 

Any thoughts?

Thank you very much for your help!

**Comment by hypery2k** [comment]
so basically you're talking about two bugs, right?

**Comment by sankar4n** [comment]
Yeah, you are correct. one is when we create a hotspot it comes up with old name and the other issue is with the configure method.

**Comment by hypery2k** [comment]
so for justing starting use `startHotspot` which just enables the hotspot again as it was set before, `createHotspot` allows custom settings, see https://github.com/hypery2k/cordova-hotspot-plugin/blob/master/PLUGIN_USAGE.MD#cordova.plugins.hotspot+createHotspot

If it's still failing please open a new issue for that (the create-part)

I've published a new version (1.0.1)
